submissions/xT0PERx/js-dom/visuality.css

```diff
@@ -0,0 +1,79 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  font-family: 'Courier New', Courier, monospace;
+}
+
+.menu-list {
+  list-style-type: none;
+  padding: 5px 0px 5px 0px;
+  background-color: rgb(248, 248, 248);
+  min-width: 100px;
+}
+
+.menu-title {
+  padding: 20px 20px 20px 20px;
+  border: solid 0.3em;
+  background-color: #e0ac1c;
+  border-color: silver;
+  border-radius: 50px 0 0 50px;
+  font-weight: bold;
+  margin: 1em;
+}
+
+.menu-item-1,
+.menu-item-2,
+.menu-item-3,
+.menu-item-4,
+.menu-item-5 {
```

Comment:
You can create a general class for all these elements and not duplicate it here

Reply:
But then my js code loses its meaning, because in my "contentArr" array, each "nameClassFromArr" key corresponds to the name of my "li" element.